Tools/autotest/pysim/util.py

```diff
@@ -329,73 +360,75 @@ def BodyRatesToEarthRates(dcm, gyro):
 
     (phi, theta, psi) = dcm.to_euler()
 
-    phiDot   = p + tan(theta)*(q*sin(phi) + r*cos(phi))
-    thetaDot = q*cos(phi) - r*sin(phi)
+    phiDot   = p + tan(theta) * (q * sin(phi) + r * cos(phi))
+    thetaDot = q * cos(phi) - r * sin(phi)
     if fabs(cos(theta)) < 1.0e-20:
         theta += 1.0e-10
-    psiDot   = (q*sin(phi) + r*cos(phi))/cos(theta)
+    psiDot   = (q * sin(phi) + r * cos(phi)) / cos(theta)
     return Vector3(phiDot, thetaDot, psiDot)
 
-radius_of_earth = 6378100.0 # in meters
+radius_of_earth = 6378100.0  # in meters
+
 
 def gps_newpos(lat, lon, bearing, distance):
-    '''extrapolate latitude/longitude given a heading and distance 
-    thanks to http://www.movable-type.co.uk/scripts/latlong.html
-    '''
+    """Extrapolate latitude/longitude given a heading and distance
+    thanks to http://www.movable-type.co.uk/scripts/latlong.html .
+    """
     from math import sin, asin, cos, atan2, radians, degrees
-    
+
     lat1 = radians(lat)
     lon1 = radians(lon)
     brng = radians(bearing)
-    dr = distance/radius_of_earth
-    
-    lat2 = asin(sin(lat1)*cos(dr) +
-                cos(lat1)*sin(dr)*cos(brng))
-    lon2 = lon1 + atan2(sin(brng)*sin(dr)*cos(lat1), 
-                        cos(dr)-sin(lat1)*sin(lat2))
+    dr = distance / radius_of_earth
+
+    lat2 = asin(sin(lat1) * cos(dr) +
+                cos(lat1) * sin(dr) * cos(brng))
+    lon2 = lon1 + atan2(sin(brng) * sin(dr) * cos(lat1),
+                        cos(dr) - sin(lat1) * sin(lat2))
```

Comment:
Can we put these two in the same line? I think it would be easier to read, don't you?